ios/app/app/API/DefaultClient.swift

```diff
@@ -79,4 +80,13 @@ class DefaultClient: Client {
             }
         }
     }
+    
+    func callAF<DataType>(for endpoint: Endpoint<DataType>, completion: @escaping (DataResponse<Any, AFError>) -> Void) {
```

Comment:
why we need this? it's basically the same as `func call` just use Alamofire to fire the request. if you prefer AF over URLSession, you can replace the implementation of `func call()` method, I'm ok with it.

secondly, I do not recommend use JSON response over Codable + Result type. I can't think any use case where they need raw JSON data instead of Models that we already provided.

Reply:
I agree that Codable + Result >>> raw JSON. I added Alamofire since, we discussed it initially but I don't think we need it since we already have URLSession and the benefit of adding Alamofire won't be needed here. I'll remove it.